gr1d99/booze_fans
# Alcohol controller create action

#### What does this PR do?
- enable api consumers to create alcohol objects into the database

#### Description of Task to be completed?
- add `create` controller action.

#### How should this be manually tested?
- checkout into branch `ft-create-alcohol`.
- run the server 'rails s`
- open `postman`, if you dont have one [click here](https://www.getpostman.com/apps)
- enter endpoint url `http://localhost:3000/alcohol/create` and provide required parameters. `name` and `vv`
- make `POST` requests and try to omit some parameters in order to see varios responses.

#### What are the relevant pivotal tracker stories?
N/A

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1139" alt="screen shot 2018-06-17 at 12 47 35" src="https://user-images.githubusercontent.com/15877982/41506767-b9091572-722c-11e8-8bb8-a5b53418e094.png">
<img width="1121" alt="screen shot 2018-06-17 at 12 47 48" src="https://user-images.githubusercontent.com/15877982/41506768-bb33ce64-722c-11e8-8ca8-407fd39df9a5.png">
<img width="1138" alt="screen shot 2018-06-17 at 12 47 59" src="https://user-images.githubusercontent.com/15877982/41506769-bceaa188-722c-11e8-9d17-29ba146a1c73.png">
